test_server.py needs pypytools, which is not available on Alpine

--- a/cheroot/test/test_server.py
+++ b/cheroot/test/test_server.py
@@ -16,6 +16,7 @@
 import requests_unixsocket
 import six
 
+pypytest = pytest.importorskip("pypytools")
 from pypytools.gc.custom import DefaultGc
 from six.moves import queue, urllib
 
